code/modules/cargo/bounties/mining.dm

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
There's no need to comment out the entire file.

Reply:
Alright. Would I just disable it from generating as a bounty in `bounty.dm` and leave everything else be?